v3/fleet-local/worker/mesos-slave@.service

```diff
@@ -19,10 +19,10 @@ ExecStartPre=/usr/bin/systemctl is-active update-os.service
 # condition / assertion options here for some reason do not prevent the
 # ExecStart from spinning up and mounting / creating .dockercfg/ as a dir
 ExecStartPre=/usr/bin/bash -c "if [ ! -f /home/core/.dockercfg ]; then exit 1; fi"
+ExecStartPre=/usr/bin/bash -c "if [ ! -f /opt/mesos/credentials ]; then exit 1; fi"
```

Comment:
@matthewdfuller in the case where the credentials are bad (i.e.: missing `/mesos/config/password`), this file will not exist and the cluster just wont spin up.